cdap-ui/app/directives/widget-container/widget-factory.js

```diff
@@ -175,12 +175,10 @@ angular.module(PKG.name + '.commons')
         }
       },
       'select': {
-        element: '<select></select>',
+        element: '<select-dropdown></select-dropdown>',
         attributes: {
-          'ng-model': 'model',
-          'class': 'form-control',
-          'ng-options': 'item as item for item in (myconfig.properties.values || myconfig["widget-attributes"].values)',
-          'ng-init': 'model = model.length ? model : (myconfig.properties.default || myconfig["widget-attributes"].default)'
+          'value': 'model = model.length ? model : (myconfig.properties.default || myconfig["widget-attributes"].default)',
```

Comment:
I think default value now is being set by the node-config controller, not by the widget.. this doesn't look quite right since we are preventing from selecting an empty option in the select widget